### PR TITLE
Update the links in README.md that point to the jts.ini and config.ini

### PR DESCRIPTION
**Pull request by ownrepo1s**
files in the github repository

### CODE REVIEW

**Comment by gnzsnz** [comment]
@ownrepo1s  thanks for fixing this.

could you please apply the same change on [template_README.md](https://github.com/gnzsnz/ib-gateway-docker/blob/master/template_README.md). otherwise your change will be overwriten on the next ibgateway/tws release.

on every release, i get latest ibgateway version and update README.md using template_README.md as input. so the error is coming form the template and it's been applied onto README.md.

hope it's clear. the short version is "please update [template_README.md](https://github.com/gnzsnz/ib-gateway-docker/blob/master/template_README.md) too" 😅

**Comment by ownrepo1s** [comment]
@gnzsnz Sure! Could you please double check if the links are correct? There are three config.ini.tmpl and three jts.ini.tmpl

**Review by gnzsnz — COMMENTED** [review]
@ownrepo1s  your changes look good. as i said the same change needs to be applied to template_README.md

i just did a PR to upgrade latest image. which means that your fork is out of sync.

probably the easiest way to move forward is the change template_README.md only. as it will not have any conflict with the latest PR (README.md will have conflicts with your version) once template is up to date, i can generate a new readme out of it. or we can wait for the next release to fix things.

so to recap, in your fork
- undo README changes
- apply changes on template_README.md
- i will commit changes on template_README

hope it's clear. thanks for your help

**Comment by ownrepo1s** [comment]
@gnzsnz Is this alright?

**Comment by gnzsnz** [comment]
thanks for your contribution @ownrepo1s
